en/docs/using-wso2-identity-server/email-templates.md

```diff
@@ -25,15 +25,12 @@ emails](_Customizing_Automated_Emails_) sent to the user.
 
         !!! note
         
-                **Note** : Ensure that you do not use the "\|" character in your
-                **Subject**, **Email Body** or **Email Footer** as this is not
-                supported.
-        
+            Ensure that you do not use the `|` character in your **Subject**, **Email Body**, or **Email Footer** as this is not
+                supported.         
 
 4.  Click **Save** to save your changes.
 
-**Recommended reading**
-
--   For an example of how email templates are used, see [Creating Users
+!!! info "Recommended reading" 
+    For an example of how email templates are used, see [Creating Users
     Using the Ask Password
-    Option](_Creating_Users_Using_the_Ask_Password_Option_).
+    Option](../../using-wso2-identity-server/creating-users-using-the-ask-passwordOption_).
```

Comment:
please rremove the underscore at the end.

Reply:
Done